src/core/server/rendering/views/fonts.tsx

```diff
@@ -532,9 +531,7 @@ export const Fonts: FunctionComponent<Props> = ({ url, theme }) => {
       url('${url}/fonts/inter_ui/Inter-UI-italic.var.woff2') format('woff2');
   }
   */
-  const fontText = theme === 'v7' ? interUi : sourceSans3;
-  const fontCode = theme === 'v7' ? roboto : sourceCodePro;
-  const fontsDefinitionRules = [fontText, fontCode]
+  const fontsDefinitionRules = [interUi, sourceSans3, roboto, sourceCodePro]
```

Comment:
@ashwin-pc I believe the goal of this change was to not bother filtering the definitions of the fonts here and allow each theme to use which ever they want. It doesn't add too much burden considering this would be compressed in transit to the browser but I don't like it much.